spec/i18next_plurals_spec.cr

```diff
@@ -0,0 +1,214 @@
+require "spectator"
+require "../src/invidious/helpers/i18next.cr"
+
+Spectator.configure do |config|
+  config.fail_blank
+  config.randomize
+end
```

Comment:
Could you require spec_helper instead of this? Youd probably just need to add a require for the helper

Reply:
The problem of the spec helper is that it `require`s the whole invidious source code. I'd prefer to keep that module somewhat separated (I'd like this to be part of @syeopite's lens library in the future).